pkg/api/defaults-kubelet.go

```diff
@@ -143,10 +143,6 @@ func (cs *ContainerService) setKubeletConfig() {
 		setMissingKubeletValues(cs.Properties.MasterProfile.KubernetesConfig, o.KubernetesConfig.KubeletConfig)
 		addDefaultFeatureGates(cs.Properties.MasterProfile.KubernetesConfig.KubeletConfig, o.OrchestratorVersion, "", "")
 
-		if cs.Properties.MasterProfile.IsUbuntu1804() {
```

Comment:
I'm a little confused why we'd remove these Ubuntu 18.04-specific defaults?

Reply:
not removed, moved to defaults.go